components/builder-depot/src/server.rs

```diff
@@ -1006,6 +1049,7 @@ pub fn router(depot: Depot) -> Result<Chain> {
 
         package_search: get "/pkgs/search/:query" => search_packages,
         packages: get "/pkgs/:origin" => list_packages,
+        packages_unique: get "/pkgs-unique/:origin" => list_unique_packages,
```

Comment:
I might suggest something a little more "restful' here for the URI. A header/param to signal that we want a unique listing or just going with `/:origin/pkgs` might be better. What do you think?